.github/workflows/tests-smoke-ipv6.yaml

```diff
@@ -133,8 +133,7 @@ jobs:
         # file (EOF) on stdin and displaying no flows.
         shell: 'script -q -e -c "bash --noprofile --norc -eo pipefail {0}"'
         run: |
-          version="$(curl -sL0 'https://api.github.com/repos/cilium/hubble/releases/latest' | jq -r .tag_name)"
-          curl -sLO "https://github.com/cilium/hubble/releases/download/${version}/hubble-linux-amd64.tar.gz"
+          curl -sLO https://github.com/cilium/hubble/releases/latest/download/hubble-linux-amd64.tar.gz
```

Comment:
If we actually want to download the latest stable Hubble CLI, I recommend we go the `stable.txt` here.

Reply:
What's the benefit?

Reply:
Just saw your message below: right now it is the case that `latest` is always the latest stable, but I was under the impression we intended have stable and patch releases in Hubble CLI and Cilium CLI in the future.

Reply:
AFAIK Hubble CLI only maintains one stable branch now, currently `v0.8` /cc @rolinh to confirm

So I guess either approach is fine, but we might want to check the `sha256sum` of the downloaded tarball in any case.

Reply:
Another thought: one advantage of using `stable.txt` would be that it matches what we suggest to our users in the Cilum docs.

Reply:
> I was under the impression we intended have stable and patch releases in Hubble CLI and Cilium CLI in the future.

As long as we have a single stable release at any given point, that wouldn't be an issue.

I'm fine with either approach. I'll switch to using `stable.txt` just because that gives us a bit more control over what version is executed.

> we might want to check the sha256sum of the downloaded tarball in any case.

That's IMO a different concern (and we should make sure we do that for all tarball, not just the two touched in this PR).